packages/carequality-sdk/src/client/carequality-fhir.ts

```diff
@@ -0,0 +1,201 @@
+    }
+    this.maxBackoff = options?.maxBackoff
+      ? dayjs.duration(options.maxBackoff, "seconds").asMilliseconds()
+      : DEFAULT_MAXIMUM_BACKOFF.asMilliseconds();
+
+    this.api = axios.create({
+      timeout: options?.timeout ?? DEFAULT_AXIOS_TIMEOUT.asMilliseconds(),
+      baseURL: baseUrl,
+      httpsAgent: this.httpsAgent,
+    });
+
+    // TODO: #1536 - improved retry logic. Issue: https://github.com/metriport/metriport-internal/issues/1536
+    axiosRetry(this.api, {
+      retries: options?.retries ?? DEFAULT_MAX_RETRIES,
+      retryDelay: retryCount => {
+        const exponentialDelay = Math.pow(2, Math.max(0, retryCount - 1));
+        const jitter = Math.random();
+        const delayWithJitter = (exponentialDelay + jitter) * BASE_DELAY.asMilliseconds();
+        return Math.min(delayWithJitter, this.maxBackoff);
+      },
+      retryCondition: error => {
+        return error.response?.status !== 200 && error.response?.status !== 403; // As per the Carequality Implementation Guide: https://carequality.org/healthcare-directory/operational_best_practices.html#reliability
+      },
+    });
+
+    this.apiKey = apiKey;
+  }
+
+  buildHeaders(headers?: Record<string, string>): Record<string, string> {
+    return {
+      ...headers,
+      "Accept-Encoding": "gzip",
+    };
+  }
+
+  async sendGetRequest(url: string, headers?: Record<string, string>): Promise<AxiosResponse> {
+    return this.api.get(url, { headers: this.buildHeaders(headers) });
+  }
+
+  async sendPostRequest(
+    url: string,
+    data: unknown,
+    headers?: Record<string, string>
+  ): Promise<AxiosResponse> {
+    return this.api.post(url, data, { headers: this.buildHeaders(headers) });
+  }
+
+  async sendPutRequest(
+    url: string,
+    data: unknown,
+    headers?: Record<string, string>
+  ): Promise<AxiosResponse> {
+    return this.api.put(url, data, { headers: this.buildHeaders(headers) });
+  }
+
+  /**
+   * Lists the indicated number of organizations.
+   *
+   * @param count Optional, number of organizations to fetch. Defaults to 1000.
+   * @param start Optional, the index of the directory to start querying from. Defaults to 0.
+   * @param oid Optional, the OID of the organization to fetch.
+   * @param active Optional, indicates whether to list active or inactive organizations. Defaults to true.
+   * @returns
+   */
+  async listOrganizations({
+    count = MAX_COUNT,
+    start = 0,
+    oid,
+    active,
+  }: ListOrganizationsParams): Promise<Organization[]> {
+    if (count < 1 || count > MAX_COUNT) {
+      throw new Error(`Count value must be between 1 and ${MAX_COUNT}`);
+    }
+
+    const query = new URLSearchParams();
+    query.append("apikey", this.apiKey);
+    query.append("_count", count.toString());
+    start !== undefined && query.append("_start", start.toString());
+    oid !== undefined && query.append("_id", oid);
+    active !== undefined && query.append("active", active.toString());
```

Comment:
nit: use `!=` so its more flexible with `null` and clients using pure JS

Reply:
Done.